packages/realm/src/app-services/App.ts

```diff
@@ -201,7 +217,7 @@ export class App<FunctionsFactoryType = DefaultFunctionsFactory, CustomDataType
         defaultRequestTimeoutMs: timeout ? binding.Int64.numToInt(timeout) : undefined,
       },
       {
-        baseFilePath: fs.getDefaultDirectoryPath(),
+        baseFilePath: baseFilePath || fs.getDefaultDirectoryPath(),
```

Comment:
Do you actually need to pass `fs.getDefaultDirectoryPath()`? I mean, core knows about this, so doesn't it default to that if `undefined` is passed as `baseFilePath`?